rskj-core/src/main/java/co/rsk/Start.java

```diff
@@ -27,16 +28,19 @@
  * The entrypoint for the RSK full node
  */
 public class Start {
-
     private static final Logger logger = LoggerFactory.getLogger("start");
-
     public static void main(String[] args) {
         setUpThread(Thread.currentThread());
 
+        RskCli rskCli = new RskCli();
+        int exitCode = rskCli.load(args);
```

Comment:
so the args are being parsed twice, aren't they? in here and second time in the `RskContext`'s ctor?

Reply:
yep, same comments as before. I'll fix all those in next commit.